diff --git a/asterixdb/asterix-app/src/test/resources/cc-main.conf b/asterixdb/asterix-app/src/test/resources/cc-main.conf
index 1837e8e52a..30b1e0bcfb 100644
--- a/asterixdb/asterix-app/src/test/resources/cc-main.conf
+++ b/asterixdb/asterix-app/src/test/resources/cc-main.conf
@@ -19,18 +19,18 @@
 txn.log.dir=target/tmp/asterix_nc1/txnlog
 core.dump.dir=target/tmp/asterix_nc1/coredump
 iodevices=target/tmp/asterix_nc1/iodevice1
-iodevices=target/tmp/asterix_nc1/iodevice2
+; iodevices=target/tmp/asterix_nc1/iodevice2
 ; iodevices=../asterix-server/target/tmp/asterix_nc1/iodevice2
 nc.api.port=19004
 #jvm.args=-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=5006
 
-[nc/asterix_nc2]
-ncservice.port=9091
-txn.log.dir=target/tmp/asterix_nc2/txnlog
-core.dump.dir=target/tmp/asterix_nc2/coredump
-iodevices=target/tmp/asterix_nc2/iodevice1
-iodevices=target/tmp/asterix_nc2/iodevice2
-nc.api.port=19005
+; [nc/asterix_nc2]
+; ncservice.port=9091
+; txn.log.dir=target/tmp/asterix_nc2/txnlog
+; core.dump.dir=target/tmp/asterix_nc2/coredump
+; iodevices=target/tmp/asterix_nc2/iodevice1
+; iodevices=target/tmp/asterix_nc2/iodevice2
+; nc.api.port=19005
 #jvm.args=-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=5007
 
 [nc]
